CurrencyDotNet.RestClient/CurrencyRestClient.cs

```diff
@@ -162,5 +157,30 @@ public void UpdatePosition()
         {
             throw new NotImplementedException();
         }
+        /// <summary>
+        /// Returns tickers for all the symbols for the last 24 hours
+        /// </summary>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public async Task<CallResult<IEnumerable<Ticker>>> GetTickersAsync(
```

Comment:
If I were to use this method, I would easily make the mistake of choosing it because of the name of the method. Maybe change to `GetTickerListAsync()` or `GetTickerCollectionAsync()`? Exceptionally personal opinion

Reply:
Fixed.